qemu/tests/x86_cpu_flag_disable.py

```diff
@@ -0,0 +1,49 @@
+import logging
+
+from virttest import error_context, env_process
+from qemu.tests.x86_cpu_model import check_flags
```

Comment:
The maintainer group now encourage to abstract the common part of multiple cases to tp-qemu/provider/, calling functions from other test cases in tp-qemu is not accepted.

Reply:
Has sent one patch to fix this. [tp-qemu/2409](https://github.com/autotest/tp-qemu/pull/2409)